sql/ts/src/skdb_orchestration.ts

```diff
@@ -168,7 +178,11 @@ function encodeProtoMsg(msg: ProtoMsg): ArrayBuffer {
       return buf.slice(0, paramsOffset + 2 + (encodeResult.written || 0));
     }
     case "tailBatch": {
-      const buffers = msg.requests.map((req) => encodeProtoMsg(req));
+      const buffers = msg.requests.map((req) => {
+        const td = new TextDecoder();
```

Comment:
td isn't used?

Reply:
Ahh, good catch!  Was using that to debug.